sensor/README.md

```diff
@@ -1,103 +1,385 @@
+### Installation alongside autoware
 
-$T(\text{sensor⎽kit⎽base⎽link}, \text{lidar1⎽base⎽link}) = T(\text{sensor⎽kit⎽base⎽link}, \text{lidar0⎽base⎽link}) \times T(\text{lidar0⎽base⎽link}, \text{lidar1⎽base⎽link})$
+After installing [autoware](https://github.com/tier4/autoware) (please see [source-installation](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/) page), execute the following commands:
 
-$T(\text{sensor⎽kit⎽base⎽link}, \text{camera0/camera⎽link}) = T(\text{sensor⎽kit⎽base⎽link}, \text{lidar0⎽base⎽link}) \times T(\text{lidar0⎽base⎽link}, \text{camera0/camera⎽link})$
+```bash
+cd autoware
+wget https://raw.githubusercontent.com/tier4/CalibrationTools/tier4/universe/calibration_tools.repos
+vcs import src < calibration_tools.repos
+rosdep install -y --from-paths src --ignore-src --rosdistro $ROS_DISTRO
+colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+```
 
-where the $T(\text{base⎽link}, \text{sensor⎽kit⎽base⎽link})$ is usually provided by a CAD model or can be simply approximated since it is a convenient frame and does not affect other computations.
+### Standalone installation (for non-autoware users)
 
-Looking at the diagram from Figure 1, we could also directly calibrate all the sensors with respect to the `base_link`. However, we believe that sensor-sensor calibration provides more accurate and consistent results so we only use one `base_link` to sensor calibration and from then all other calibrations are performed via pairs of sensors.
+The sensor calibration tools are usually used as part of the autoware ecosystem. However, they can also be used for projects outside autoware, or even outside autonomous driving. Note: due to its use in autoware, even if it is possible to use the sensor calibration tools independently, due to some light dependencies, parts of autoware still need to be downloaded, even if they are not all compiled.
 
-### Generic calibration
+The following commands present an example of how to install the calibration tools and their dependencies assuming you have a ROS2 workspace called `workspace`:
 
-- [Extrinsic manual calibration](docs/how_to_extrinsic_manual.md)
+```bash
+# Install vcs (if needed, follow the instructions from https://github.com/dirk-thomas/vcstool)
+sudo apt-get install python3-vcstool
 
-Intended as a proof-of-concept of our calibration API and as a baseline to which to compare automatic calibration tools, this method allows us to directly modify the values of the TF tree with a rviz view to evaluate the tfs and calibration.
+# Download the calibration tools and its dependencies
+cd workspace
+wget https://raw.githubusercontent.com/tier4/CalibrationTools/tier4/universe/calibration_tools_standalone.repos
+vcs import src < calibration_tools_standalone.repos
```

Comment:
vcs import src < calibration_tools_standalone.repos stuck.

Reply:
In addition to say `stuck`, please identify the reasons behind it.
For example, it could have been your conection, keys, wrong links, etc